src/main/java/nl/knaw/huc/resussun/resources/RootResource.java

```diff
@@ -86,32 +78,15 @@ private ServiceManifest createServiceManifest() {
   private Map<String, Candidates> search(Map<String, Query> queries) throws IOException {
     final Map<String, Candidates> candidates = new HashMap<>();
 
-    try (final RestHighLevelClient elasticsearchClient = elasticSearchClientFactory.build()) {
+    try (final SearchClient searchClient = searchClientFactory.createSearchClient()) {
       for (Map.Entry<String, Query> querySet : queries.entrySet()) {
         final String field = querySet.getKey();
         final String queryText = querySet.getValue().getQuery();
 
-        final SearchSourceBuilder query =
-                new SearchSourceBuilder().query(QueryBuilders.queryStringQuery("*" + queryText + "*").queryName(field));
-        final SearchResponse response =
-                elasticsearchClient.search(new SearchRequest("index").source(query), RequestOptions.DEFAULT);
-
         final Candidates results = new Candidates();
         candidates.put(field, results);
 
-        for (final SearchHit hit : response.getHits()) {
-          final JsonNode source = objectMapper.readTree(hit.getSourceAsString());
-          final String type = source.get("rdf_type").get("title").get("value").asText();
-
-          final Candidate candidate = new Candidate(
-                  hit.getId(),
-                  source.get("title").get("value").asText(),
-                  hit.getScore() * 100,
-                  false
-          ).type(type, type);
-
-          results.candidate(candidate);
-        }
+        searchClient.search(queryText, field, results::candidate);
```

Comment:
Good idea to move the actual searching out of the controller. However:
1. We might want to give the search client the Query object instead of just the query text. The query object also includes other information (like a limit) that we might want to implement later. 
1. The search client does not need the field right?
1. I like the consumer for the candidate that is returned. However, it is rather obscure right now, so the Candidates class needs to be refactored. Probably just a constructor for the field and the 'candiate' method should be renamed to 'addCandidate' or something?